src/llmq/quorums_blockprocessor.cpp

```diff
@@ -303,16 +299,16 @@ bool CQuorumBlockProcessor::IsCommitmentRequired(Consensus::LLMQType llmqType, c
 }
 
 // WARNING: This method returns uint256() on the first block of the DKG interval (because the block hash is not known yet)
-uint256 CQuorumBlockProcessor::GetQuorumBlockHash(Consensus::LLMQType llmqType, const CBlockIndex* pindexPrev)
+uint256 CQuorumBlockProcessor::GetQuorumBlockHash(Consensus::LLMQType llmqType, const CBlockIndex* pindex)
 {
     auto& params = Params().GetConsensus().llmqs.at(llmqType);
 
-    int nHeight = pindexPrev->nHeight + 1;
+    int nHeight = pindex->nHeight;
     int quorumStartHeight = nHeight - (nHeight % params.dkgInterval);
-    if (quorumStartHeight >= pindexPrev->nHeight) {
```

Comment:
This seems like a logic change, but wanted to make sure this is intentional if so. Before this change, in the case that `quorumStartHeight` is greater than or equal to previous index, it returns empty uint256, but post-change this is checking for `quorumStartHeight` greater than current index.

Is this intentional, or am I missing something?

Reply:
oops, I got this one wrong 🙈 fixed :)